tests/cli/dispatchers/test_messages.py

```
@@ -199,6 +201,43 @@ async def test_process_update_chunk_renders_tool_message(self, mock_session):
 
+    @pytest.mark.asyncio
+    async def test_stream_response_handles_cancel_and_cleans_up(
+        self, mock_session, mock_context
+    ):
+        """Cancelled streaming should finalize output, reset prompt, and clear task."""
+
+        dispatcher = MessageDispatcher(mock_session)
+        mock_session.context = mock_context
+
+        # Fake streaming generator that immediately raises CancelledError
+        async def cancelled_stream(*_args, **_kwargs):
```

Ruff (0.14.7)

Missing return type annotation for private function `cancelled_stream`

(ANN202)